canonical-web-and-design/ubuntu.com
# Update text of /telecommunications page

## Done

- A very large update to /telecommunications

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/telecommunications
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [copy doc](https://docs.google.com/document/d/15FY_IaT39V81FQcoSoEJpmRRjBX80PJ7uxjgzHnDDws/edit#heading=h.t5nt9uvsqry)
- Provide design guidence - images, layout, double quotes, etc...

## Issue / Card

Fixes #5756
